packages/graphback-cli/src/templates/resources/config/mongodb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "user": "mongodb",
+  "password": "mongo",
+  "database": "users",
```

Comment:
Graphback?

Reply:
Name the database graphback?

Reply:
Yes, but we do not need this now since migrate is gone